src/operator/nn/dropout.cc

```diff
@@ -163,6 +161,9 @@ Example::
 #endif
     }
     request.emplace_back(ResourceRequest::kParallelRandom);
+#if MXNET_USE_MKL_DROPOUT
```

Comment:
is this a new env variable?

Reply:
No, it has been already defined in `dropout-inl.h`.

Reply:
In original implementation, this env variable will be disabled in the end of `dropout-inl.h` (see https://github.com/apache/incubator-mxnet/blob/master/src/operator/nn/dropout-inl.h#L532).

Reply:
thanks for the explanation.